### PR TITLE
Add release notes for 8.9.0

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

File: _drafts/2020-11-05-vowpalwabbit-8.9.0.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+
+It's been a while since we last released, but a lot of exciting things have been going on! Let's take a look at some of the big items from this release.
+
+## Python Wheels
+
+We now produce Python wheels for [most platforms](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Python#support) which are uploaded to PyPi. This means you almost never have to build from source to use VW in Python! We're committed to making using VW in Python a great experience, and this is just the first step.
+
+## Initial Pandas support in Python
+
+@lalo to fill in
```
Comment (lalo) [inline _drafts/2020-11-05-vowpalwabbit-8.9.0.md:14]:
```suggestion
We introduced in this release `DFtoVW`, a class that converts `pandas.DataFrame` object into VW input format. To use import it from `vowpalwabbit.DFtoVW`. We currently support simple label, multiclass label and feature transformations.

For more information be sure to check out 

- https://github.com/VowpalWabbit/vowpal_wabbit/blob/80917701fb0353f8f862cab3a0f8425bab37396c/python/vowpalwabbit/DFtoVW.py#L517 
- https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Input-format
- https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/python/tests/test_DFtoVW.py


```

**Review by olgavrou — COMMENTED** [review]
This is great and reads nicely!

Do we want to have a list of resolved bugs?

**Comment by jackgerrits** [comment]
Resolved bugs would be nice, I'm not quite sure how to identify all of them from the changelist though

**Comment by cheng-tan** [comment]
This looks awesome! Do you want to update the banner message as well?

**Pull request merged**